config/env_sample.hjson

```diff
@@ -95,8 +95,6 @@
     # strings for construct file download url
     "CANVAS_FILE_PREFIX": "https://umich.instructure.com/files/",
     "CANVAS_FILE_POSTFIX": "/download?download_frd=1",
-    # Paths to Google BigQuery credential json file
-    "GOOGLE_APPLICATION_CREDENTIALS": "/secrets/bq_cred.json",
```

Comment:
Is the "GOOGLE_APPLICATION_CREDENTIALS" param no longer needed?

Reply:
We do need it, I just don't think it's being used when you define it in `env.hjson`, only in `.env`. I think the BigQuery library uses an environment variable (https://cloud.google.com/bigquery/docs/reference/libraries), and unless I'm mistaken we never add values from `env.hjson` to the environment (we're kinda doing the opposite here). For deployments, `GOOGLE_APPLICATION_CREDENTIALS` seems to be set in the environment in our Jenkins pod definitions.

I removed this here because in my testing, the needed value in `.env` was overriding the value. It was doing any harm, but it was creating some unnecessary noise. I also figured it could lead to some confusion if someone thought it needed to be updated here but it wasn't doing anything.

Reply:
Sounds good. Better not to keep the redundant definition.